manifests/04_roles.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:openshift:operator:cert-manager
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+rules:
+  - apiGroups:
+    - ""
+    resources:
+      - services
+      - events
+      - configmaps
+      - secrets
+    verbs:
+      - list
+      - get
+      - create
+      - patch
+      - update
+      - watch
+      - delete
+  - apiGroups:
+      - apps
+    resources:
+      - deployments
+    verbs:
+      - list
+      - get
+      - create
+      - update
+      - watch
```

Comment:
Those roles are required by the Operator to create the operand. It's not a complete list and it will evolve. Currently there's no easy way to try it out. We need to have the Operator images first and let it run with its Service Account. Then we'll see where it fails.

Reply:
don't add the roles until you need them, then

Reply:
I adjusted the manifests to what we need (at least in theory). Once we have our images built, I'll test them.